src/csv/area_and_circum_calculations.py

```diff
@@ -0,0 +1,27 @@
+import math
+
+
+def area_of_circles(radius):
+    area = (math.pi * radius) * (math.pi * radius)
+    return area
+
+
+def circ_of_circles(radius):
+    circ = 2 * math.pi * radius
+    return circ
+
+
+def do_build_data(current_position, line_array):
+    build_data = ""
+
+    if current_position > 1:
+        name = line_array[0]
+        radius = int(line_array[1])
+
+        a = " area = "
+        c = " circumference = "
```

Comment:
Try not to use abbreviated variable naming like c. Instead using real naming conventions e.g. circumference.

Reply:
fixed